en/engine/event/event-node.md

```diff
@@ -0,0 +1,244 @@
+| __Node.EventType.MOUSE_UP__    | When a button on the mouse is released                                                       |
+| __Node.EventType.MOUSE_WHEEL__ | When the mouse wheel is scrolled                                                             |
+
+The important APIs of mouse events (`Event.EventMouse`) are described in the [Mouse Events API](event-api.md#Mouse-Event-API) (`Event` standard events API excluded).
+
+## Touch event types and event objects
+
+__Touch events__ can be triggered on both mobile platforms and desktop platforms. Developers can better debug on the desktop platform, by simply listening for touch events and responding to both mobile touch events and desktop mouse events at the same time. The types of touch events provided by the system are as follows:
+
+| Enumerated Object Definition  | Timing of Event Triggering                                                               |
+|:--------------------------------|:-----------------------------------------------------------------------------------------|
+| __Node.EventType.TOUCH_START__  | When one or more touch points are placed in the target node region                       |
+| __Node.EventType.TOUCH_MOVE__   | When one or more touch points are moved along the screen                                 |
+| __Node.EventType.TOUCH_END__    | When one or more touch points are removed from the screen in the target node region      |
+| __Node.EventType.TOUCH_CANCEL__ | When one or more touch points are removed from the screen outside the target node region |
+
+The important APIs of a touch event (`Event.EventTouch`) are described in the [Mouse Events API](event-api.md#Touch-Event-API) (`Event` standard event API excluded):
+
+> __Note__: touch events support multi-touch, each touch spot will send one event to the event listener.
+
+## Node Event Dispatching
+
+We support `dispatchEvent` interface on `Node`. Events dispatched by this interface, would enter the event delivery stage. The event dispatching system of Cocos Creator is based on the implementation of [event bubbling and capture on Web standard](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Building_blocks/Events#event_bubbling_and_capture). After the event is dispatched, it will go through the following three stages:
+- __capturing phase__: The event is passed from the scene root node to the child nodes step by step, until it reaches the target node or the event propagation is stopped in the event callback
+- __target phase__: The event is triggered on the target node
+- __bubbling phase__: The event is bubbled from the target node to the parent node level by level, until the root node is reached or the event propagation is stopped in the event callback
+
+When we call `node.dispatchEvent()`, it means that `node` is the target node mentioned above. In the process of event delivery, we can stop the event propagation by calling `event.propagationStopped = true`.
```

Comment:
```suggestion
When calling `node.dispatchEvent()`, it means that `node` is the target node mentioned above. In the process of event delivery, to stop the event propagation by call `event.propagationStopped = true`.
```

Reply:
I change this way https://github.com/cocos-creator/creator-docs/pull/1903/commits/0a6ac9d9c7777a7d03720b2cb905a237de48cf91 https://github.com/cocos-creator/creator-docs/pull/1903/commits/0c3fe09cb338f552782865ac09f2bb447b29d885